### PR TITLE
Indoor get

### PR DESCRIPTION
**Pull request by PierreGoe**

### CODE REVIEW

**Review by Mariion-62 — APPROVED** [review]
it's okay

File: index.js [hunk 1]
```diff
@@ -7,9 +7,26 @@ app.use(express.json());
...
   try {
-    const rows = await db.query(`SELECT * FROM mesures;`);
-    await res.status(200).send(rows);
+    db.query(
+      `SELECT distinct adresses_longitude, adresses_latitude FROM mesures;`
+    ).then((data) => {
+      data[0].map((row) => {
+        return db
+          .query(
+            `SELECT * FROM mesures WHERE adresses_longitude=? AND adresses_latitude=? ORDER BY timestamp LIMIT 1;`,
```
Comment (Hacene010) [inline index.js:18]: be aware about the select *

File: index.js [hunk 2]
```diff
@@ -7,9 +7,26 @@ app.use(express.json());
...
+            [row.adresses_longitude, row.adresses_latitude]
+          )
+          .then((data2) => {
+            result.push(data2[0]);
+            console.log(result.length);
```
Comment (Hacene010) [inline index.js]: no console log !!!

**Review by Hacene010 — CHANGES_REQUESTED** [review]
remove the console log plz

**Review by Hacene010 — APPROVED** [review]

**Pull request merged**